### PR TITLE
Todo list interactive

### PR DESCRIPTION
**Pull request by roniy68**
# Highlights 🥇 

## What I Have Done: 
- Added Functions :✅ 
  * completed ? true : false
- Added Event listener to the checkbox ✅ 
- Clear all completed button works✅ 
- Local storage updation ✅ 

### [ Live Here ](https://roniy68.github.io/to-do-list/)

deploy 

## Followed Rules: 
- Made sure [no linter errors](https://github.com/microverseinc/linters-config)
-  Followed [Github Flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md)
-  Documented [professionaly](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)
-  Followed the list of [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).
-  Followed our list of [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md).

### CODE REVIEW

File: src/modules/eventListeners.js [hunk 1]
```diff
@@ -39,18 +39,21 @@ todoList.addEventListener('click', (e) => {
...
 });
 
+// Add event listener for double click to edit
 todoList.addEventListener('dblclick', (e) => {
   if (e.target.closest('.todo-list-li-text')) {
     editTask(e, toDoTasks);
   }
 });
 
+// Delete a task on click to the cross button
```
Comment (roshan-bajgain) [inline src/modules/eventListeners.js:56]: - [x] According to the project requirement, the updated array of items should be stored in local storage but the index is repeating. Please fix this issue.
- [x] I notice that when I remove the completed list and then add the new one then it shows the list which is removed already. Please fix this issue. 
- [x] You can solve this issue by updating the local storage after the user does `clear all completed`. You did not update the clear all button on local storage so it is re-rendering again.
### Error
his

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @roniy68, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by roniy68** [comment]
* storage ✅ 
* updatestorage() invoked ✅ 

🎥 watch 🎥 

[
image
](https://www.loom.com/share/0d814180cbd6464da7d80779ac740531)

**Review by AshakaE — APPROVED** [review]
Hi @roniy68 , 

### Highlights ✨ 

- Pull request 👌 
- Git flow ✅ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Proceed to the next level 👏 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**